src/cls/_ZPM/PackageManager/Client/FilesystemServerDefinition.cls

```diff
@@ -196,9 +196,9 @@ Method AddCacheItem(pModuleFileName As %String, pSubDirectory As %String, Output
 			// Contained multiple documents - tricky! We'll just continue.
 			Quit
 		}
-		
 		Set tExt = $ZConvert($Piece($Get(tLoadedList),".",*),"U")
-		If (tExt '= "ZPM") {
+		// Previously this is '= zpm. Changed it to module
+		If (tExt '= "MODULE") {
```

Comment:
Nit: Rather than hardcoded string, perhaps use one of the parameters from the studio document class which determine the extension?